ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/aliases.schema.yml

```
@@ -0,0 +1,9 @@
+# yaml-language-server: $schema=../../../../plugins/plugin_utils/schema/avd_meta_schema.json
+# Line above is used by RedHat's YAML Schema vscode extension
+# Use Ctrl + Space to get suggestions for every field. Autocomplete will pop up after typing 2 letters.
+type: dict
+keys:
+  aliases:
+    type: list
+    items:
+      type: str
```

```suggestion
  aliases:
    type: str
    description: |
      Multiline string with one or more multiple alias commands.
      Example:
      ```
      aliases: |
        alias wr copy running-config startup-config
        alias siib show ip interface brief
      ```
```